test/setup-browser-env.js

```diff
@@ -68,6 +68,11 @@ Object.defineProperty(window, 'clipboardData', {
   writable: true
 });
 
+// These do not seem to be present under jsdom v16, even though the documentation suggests that should be the case
+['addEventListener', 'removeEventListener', 'dispatchEvent'].forEach(prop => {
```

Comment:
Spent a lot of time on this one, eventually had to patch it up like so as I am not sure what the underlying issue is exactly